.werft/values.dev.yaml

```diff
@@ -37,8 +37,16 @@ components:
     wsman:
     - name: "local"
       address: "ws-manager:8080"
+      tls:
```

Comment:
IMO configuring the root certificate here would be sufficient, cmp. https://grpc.io/docs/guides/auth/#with-server-authentication-ssltls-6. That would also fit into the current `gpctl`/DB implementation where we have one `certificate` field.

Reply:
Correct me if I'm wrong but I think this is not true for mutual TLS because the root CA is just for checking the server cert but with mutual TLS the client needs a cert as well so that the server can check the client cert. (we have 2 certs here: one for server, one for client, both signed with the same CA)

See e.g. here for a brief example: https://krishicks.com/post/grpc-mutual-tls-golang/

Reply:
Absolutely true, just thought we wanted server auth + token. After offline discussion we're going with mutual TLS.